src/Simplex.jl

```diff
@@ -3,18 +3,25 @@ module Simplex
 using SparseArrays
 using LinearAlgebra
 using Statistics
+using MatrixOptInterface
 using TimerOutputs
 using CUDA
 
+const MatOI = MatrixOptInterface
+const MOI = MatOI.MOI
+
 const TO = TimerOutput()
 
+nrows(lp::MatOI.AbstractLPForm{T}) where T = size(lp.A, 1)
+ncols(lp::MatOI.AbstractLPForm{T}) where T = size(lp.A, 2)
+
 include("Constant.jl")
 include("GLPK.jl")
 include("LP.jl")
 include("PhaseOne/PhaseOne.jl")
 
 mutable struct SpxData{T<:AbstractArray}
```

Comment:
I think at some point we should update the templating of `SpxData`. I would suggest doing something like:
```julia
mutable structu SpxData{T, VT<:AbstractVector{T}, MT::AbstractArray{T, 2}} where T
```